tests/unit/customizations/test_commands.py

```diff
@@ -55,3 +55,17 @@ def test_load_subcommand_table_property(self):
         # Ensure the ``subcommand_table`` is not built again if
         # ``subcommand_table`` property is called again.
         self.assertIs(orig_subcommand_table, self.command.subcommand_table)
+
+    def test_load_lineage(self):
+        self.assertEqual(self.command.lineage, [self.command])
+
+    def test_pass_lineage_to_child_command(self):
```

Comment:
Can't we do this without patching by just creating a new command class that has subcommands instead of using the `BasicCommand` instance?  I'd prefer to avoid mocking these types of calls especially if they don't need to be.